internal/pkg/cli/flag.go

```diff
@@ -228,8 +229,8 @@ Must be one of: %s.`, strings.Join(applyAll(validIaCTools, strconv.Quote), ", ")
 	cdkLanguageFlagDescription = `Optional. The Cloud Development Kit language.`
 	overrideEnvFlagDescription = `Optional. Name of the environment to use when retrieving resources in a template.
 Defaults to a random environment.`
-
-	repoURLFlagDescription = fmt.Sprintf(`The repository URL to trigger your pipeline.
+	skipResourcesFlagDescription = `Optional. Skip asking for which resources to override and generate empty patch files.`
```

Comment:
just a dumb question: do you think after we introduce yaml patch feature this `patch` work here would be little confusing? I am not sure about it, but in first glance I got confused if I am reviewing override feature or yaml patch. It could be just me so feel free to ignore if patch is the right word that should be used here.

Reply:
👍 Good thinking. Let me rename all "patch" occurrences to "extensions".

YAMLPatch is just one way of extending, but an alternative is CDK extensions

Reply:
Done! Is that better?